### DISCUSSION TITLE
Is it possible to build drivers not included by default?

### DISCUSSION THREAD

**Discussion by rmkilc** [hassio-addons/app-nut#357] [Q&A]
There are drivers not included by default, like **generic_modbus**, **snmp-ups**, and others. These can be enabled with the following commands before calling make:
```
configure --with-modbus=yes
configure --with-snmp=yes
```
Is there a way to use one of these with this addon if needed? I am after the **generic_modbus** driver.

**Comment by TimBeekman** [comment]
+1 from me!

I need the apc_modbus driver.
